packages/editor/break/src/exit-break/transforms/exitBreak.ts

```diff
@@ -13,7 +13,7 @@ import { ExitBreakRule } from '../types';
 export const exitBreak = <V extends Value>(
   editor: PlateEditor<V>,
   {
-    level = 0,
+    level = 1,
```

Comment:
NB: `/packages/editor/break/src/exit-break/types.ts` docstring says default is 1, this was previously 0 🤷🏻

Reply:
It should be 0, doc typo

Reply:
@zbeyens 0 sounds like a sensible default for "which level should it break out at", but this PR modifies the API so that it instead means "how many layers above the current should it break out at", for which I think 1 is a better default. Appreciate that this may be a breaking change for some but otherwise it's not a useful plugin for editors with nested nodes e.g. tables 😔

Reply:
Makes sense, what about an additional option to enable this, so we make the breaking change later?

Reply:
Sounds good 👍🏻